Tests/MongoSwiftTests/MongoCollection+IndexTests.swift

```diff
@@ -218,17 +218,18 @@ final class MongoCollection_IndexTests: MongoSwiftTestCase {
     }
 
     func testCreateDropIndexByModelWithMaxTimeMS() throws {
-        let center = NotificationCenter.default
         let maxTimeMS: Int64 = 1000
 
-        let client = try MongoClient.makeTestClient(options: ClientOptions(commandMonitoring: true))
+        let client = try MongoClient.makeTestClient(options: ClientOptions(commandMonitoring: true,
```

Comment:
Because all clients were firing events with the change I made above, I needed to use a different center to filter the events properly. I've just gone ahead and reverted all the changes I made to this test.